### ISSUE TITLE
post a video using alamofire multipart/form-data do not see the post data at all

### ISSUE THREAD

**Issue by adamgf** [ProxymanApp/Proxyman#760]
### Proxyman version? (Ex. Proxyman 1.4.3)

2.16.0 using Atlantis in iPhone app

### macOS Version? (Ex. mac 10.14)

11.0.1 Beta

### Steps to reproduce 

Set up an iOS app to use Atlantis and Proxyman on mac.

Via iOS app post a valid mp4 file to Amazon S3 using iOS AlamoFire API

### Expected behavior

To see the form data in the request body!!!  Instead it says 'Empty Body'!!!

### Screenshots (optional)

<img width="1294" alt="Screen Shot 2021-01-16 at 11 06 22 PM" src="https://user-images.githubusercontent.com/1305327/104834095-236f6e80-5852-11eb-8b32-3f6b7d2f4844.png">

**Comment by NghiaTranUIT** [comment]
Look like it's a bug. I will try to reproduce and fix it 👍

**Comment by adamgf** [comment]
Cool thanks @NghiaTranUIT!  That will be really appreciated!

**Comment by adamgf** [comment]
I am not sure but I think the data may be AES256 encrypted so that may be why it is not being decrypted I do not need to see the entire multipart/form-data request body the binary data can just be stubbed out but I was more interested in seeing the key values for the other data that gets posted.

**Comment by NghiaTranUIT** [comment]
To workaround, you can use normal SSL Proxying (by installing the certificate to iPhone and enable SSL Proxying on this domain) to see the multiple-part data from these Upload Requests.

I will check out the Atlantis to see if we can get the Encrypted Body from URLSession or not

**Comment by adamgf** [comment]
The server does not accept the certificate.

**Comment by adamgf** [comment]
You should be able to get the multipart/form data out of the request

**Comment by NghiaTranUIT** [comment]
Yup, Atlantis should be able to get the multipart body from both the request and response too 😄 I will notify you when I fix soon 👍

**Comment by NghiaTranUIT** [comment]
@adamgf if you don't mind, please check out this working branch from Atlantis: https://github.com/ProxymanApp/atlantis/pull/49

You can use it by adding this Pod
```
pod 'atlantis-proxyman', :git => 'https://github.com/ProxymanApp/atlantis.git', :branch => 'feat/upload-task'
```

**Comment by NghiaTranUIT** [comment]
Just a friendly reminder that your request is implemented in Atlantis 1.6.0 🎉 

You can update normally by `pod update`, It will get the latest version 🙌
